README.md

```diff
@@ -30,16 +30,26 @@ INSTALLED_APPS = [
 ```python
 ...
 OPENID_CONNECT_VIEWSET_CONFIG = {
-    "REDIRECT_AFTER_AUTH": "http://localhost:3000",
-    "USE_SSO_COOKIE": True,
-    "SSO_COOKIE_DATA": "email",
     "JWT_SECRET_KEY": JWT_SECRET_KEY,
     "JWT_ALGORITHM": JWT_ALGORITHM,
+    "REQUIRED_USER_CREATION_FIELDS": ["email", "first_name", "username"],
+    "USER_CREATION_FIELDS": ["email", "first_name", "last_name", "username"],
+    "MAP_CLAIM_TO_MODEL": {
+        "given_name": "first_name",
+        "family_name": "last_name",
+        "preferred_username": "username",
+    },
+    "SPLIT_NAME_CLAIM": False, # Whether to split the `name` claim into first_name & last_name if present
+    "USER_UNIQUE_FILTER_FIELD": "username",
+    "USE_SSO_COOKIE": True,
+    "SSO_COOKIE_DATA": "email",
     "SSO_COOKIE_MAX_AGE": None,
     "SSO_COOKIE_DOMAIN": "localhost",
+    "JWT_ALGORITHM": "HS256",
     "USE_AUTH_BACKEND": False,
     "AUTH_BACKEND": "",  # Defaults to django.contrib.auth.backends.ModelBackend
-    "USE_RAPIDPRO_VIEWSET": False,
+    "REDIRECT_AFTER_AUTH": "http://localhost:3000",
+    "USER_UNIQUE_FILTER_FIELD": None # Defaults to username
```

Comment:
Should we be using this `None` value, or should we be using the `username`?
This also seems to be a duplicate key. It's being introduced in line 43 above

Reply:
`username` is usually the default. Let me remove this duplicate